extensions/pyRevitTools.extension/pyRevit.tab/Analysis.panel/ColorSplasher.pushbutton/script.py

```diff
@@ -588,52 +574,52 @@ def getRangeOfValues(category, param, new_view):
                 break
     copy = [x for x in list_values if x._value == "None"]
     if len(copy) > 0:
-        list_values.Remove(copy[0])    
-    #Ordenar
+        list_values.Remove(copy[0])
+    # Order
     list_values = sorted(list_values, key=lambda x: x._value, reverse=False)
-    #list_values.sort(key=lambda x: x._value, reverse=False)
     if len(list_values) > 1:
         try:
             first_value = list_values[0]._value
             indx_del = getIndexUnits(first_value)
             if indx_del ==0:
                 list_values = sorted(list_values, key=lambda x: float(x._value))
-            elif indx_del < len(first_value) and indx_del !=-1:
+            elif indx_del < len(first_value) and indx_del != -1:
                 list_values = sorted(list_values, key=lambda x: float(x._value[:-indx_del]))
         except Exception as e:
             pass
     if len(copy) > 0 and len(copy[0]._eleId) > 0:
         list_values.Add(copy[0])
     return list_values
 
+
 def getCategoriesAndParametersInUsed(cat_exc, acti_view):
-    #Get All elements and filter unneeded
-    collector = FilteredElementCollector(doc, acti_view.Id).WhereElementIsNotElementType().WhereElementIsViewIndependent().ToElements()
+    # Get All elements and filter unneeded
+    collector = DB.FilteredElementCollector(doc, acti_view.Id).WhereElementIsNotElementType().WhereElementIsViewIndependent().ToElements()
     list_cat = []
     for ele in collector:
-        if ele.Category != None:
+        if ele.Category is not None:
```

Comment:
invert if and de-indent the rest

```suggestion
        if ele.Category is None:
            continue
```